CONTRIBUTING.md

```diff
@@ -144,6 +149,9 @@ Observe the following guidelines when submitting a pull request for review
 * Ensure that every single commit passes CI — this is mandatory
 * Feel free to use Github comments to clarify implementation points and consider adding the same comments to the code
 * Feel free to use diagrams, sample code, or links to specific parts of external documentation — they are highly encouraged
+* Please respond to inline comments made in pull requests: it makes it easier to track what has been done and
```

Comment:
I wonder if it isn't easier to encourage contributors to directly resolve the threads themselves **iff** they've addressed the comments and leave it open otherwise. Seems like a bit of an overkill to respond to all threads when there's already a button to do just that, at least that's how I personally do that - either I have a counterargument for a given thread or I just address the comment and mark as resolved.
Arguably this is a matter of taste.

Reply:
"Done" + resolve could be an option. I think "done" or any other minimal-effort response is better than just resolving, at least for 3rd party contributors: this makes it obvious that they have read the comment, and at the very least typed "done" in and resolved rather than just resolving and ignoring it completely.

Reply:
fair enough, works for me.